floater/test/test_generators.py

```diff
@@ -113,5 +113,7 @@ def test_to_mitgcm_format(fs, tmpdir):
 	    assert os.path.exists(filename)
             array = np.fromfile(filename, dtype='>f4')
             assert len(array)==(fs.Nx*fs.Ny + 1 )* 9
+            npart = fs.N
```

Comment:
In line 115, we are already assuming that N=fs.Nx*fs.Ny. I guess that is not correct if we have a land mask. But then why does the following code pass?
```python
 assert len(array)==(fs.Nx*fs.Ny + 1 )* 9
```